ui/app/src/modules/Preview/QuickCreate.tsx

```diff
@@ -134,10 +163,31 @@ export function QuickCreateMenu(props: QuickCreateMenuProps) {
     if (siteId) {
       getQuickCreateContentList(siteId).subscribe(
         (data) => setQuickCreateContentList(data.items),
-        (error) => setError(error.response.response)
+        (error) => {
+          dispatch(
+            showErrorDialog({
+              error
+            })
+          );
+        }
       );
     }
-  }, [siteId]);
+  }, [siteId, dispatch]);
+
+  useEffect(() => {
+    getStudioInfo().subscribe(
+      (info) => {
+        setStudioVersion(info.version.packageVersion.substr(0, 3));
```

Comment:
Could we get this from the freemarker template? If so we could put it in state/env and retrieve there from instead.